src/tests/UnitTests/Navigation/WPF/NavigationServiceTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿using CodeMonkeys.DependencyInjection;
+using CodeMonkeys.Navigation;
+using CodeMonkeys.Navigation.WPF;
+using CodeMonkeys.UnitTests.Navigation.ViewModels;
+using CodeMonkeys.UnitTests.Navigation.WPF.Views;
+
+using Moq;
+using NUnit.Framework;
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace CodeMonkeys.UnitTests.Navigation.WPF
+{
+    public partial class NavigationServiceTests
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5575746)